clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -761,7 +790,7 @@ public void assign(List<TopicPartition> partitions) {
     }
 
     /**
-     * Fetches data for the topics or partitions specified using one of the subscribe/assign APIs. It is an error to not have
+     * Fetch data for the topics or partitions specified using one of the subscribe/assign APIs. It is an error to not have
```

Comment:
Not on this line, but the paragraph below says that the seeked position will be used on every rebalance. In fact, we always reset to the latest committed offset or using the reset strategy on rebalances.